docs/paper/dev/event-api/custom-events.md

```diff
@@ -5,11 +5,11 @@ slug: /dev/custom-events
 # Custom Events
 
 Creating custom events is a great way to add functionality to your plugin. 
-This will allow for other people to listen for your custom events and add functionality to your plugin.
+This will allow other people to listen to your custom events and add functionality to your plugin.
```

Comment:
I'd say "other plugins to listen", not players.

Reply:
It says other people, but the point is still valid, thanks :)